buildSrc/src/main/groovy/openhouse.iceberg-conventions.gradle

```diff
@@ -1,5 +1,5 @@
 ext {
-  icebergVersion = '1.2.0'
+  icebergVersion = '1.5.2'
```

Comment:
Why is name not iceberg-conventions-1.5.2

Reply:
Yeah we can change that name too.

Reply:
Cool let's do it